en/docs/develop/setting-up-user-portal-in-a-dev-environment.md

```diff
@@ -0,0 +1,101 @@
+# Setting-Up User Portal in a Development Environment
+
+In [WSO2 Identity Server 5.10.0](https://wso2.com/identity-and-access-management/) the user portal is available under
+ the URL `http(s)://<SERVER_HOST>/t/<TENANT_DOMAIN>/user-portal`. If you are planning on setting up the repo in a
+ development environment follow the steps given below.
+
+!!! note "Before you begin"
+
+    1. Install [node](https://nodejs.org/en/download/) if you haven’t already(npm is already bundled with node).
+    2. Install [maven](https://maven.apache.org/download.cgi) (Needed to run mvn commands).
+    3. A running Identity Server v 5.10. (If you want to build from source follow the instructions listed [here]
+    (https://github.com/wso2/product-is)).
+
+
+### Step 1: Configuring Identity Server
+
+!!! tip
+    Default dev origin, hostname and port set for the User Portal is https://localhost:9000. So the following
+    configurations will have to change accordingly if you change the port or any other configurations.
+
+
+1.  Allowing Cross Origin requests to the Identity Server.
+
+    Cross origin requests will be blocked by default as a security measure. So you have to put the following CORS
+    filter to the config file `<IS_HOME>/repository/resources/conf/templates/repository/conf/tomcat/web.xml.j2` in
+    WSO2 Identity Server distribution pack.
```

Comment:
Cross-origin requests are blocked by default in WSO2 IS as a security measure. Add the following CORS filter to the `<IS_HOME>/repository/resources/conf/templates/repository/conf/tomcat/web.xml.j2` file to enable it.